featurebyte/service/working_schema.py

```diff
@@ -54,54 +46,21 @@ class WorkingSchemaService(BaseService):
     WorkingSchemaService is responsible for managing the working schema in the data warehouse
     """
 
-    _task_manager: TaskManager = PrivateAttr()
-
-    def __init__(self, user: Any, persistent: Persistent, celery: Celery, catalog_id: ObjectId):
+    def __init__(
+        self,
+        user: Any,
+        persistent: Persistent,
+        catalog_id: ObjectId,
+        feature_service: FeatureService,
+        tile_scheduler_service: TileSchedulerService,
+        tile_manager_service: TileManagerService,
+        feature_manager_service: FeatureManagerService,
+    ):
         super().__init__(user, persistent, catalog_id)
-        self.table_service = TableService(user=user, persistent=persistent, catalog_id=catalog_id)
-        self.view_construction_service = ViewConstructionService(
-            user=user,
-            persistent=persistent,
-            catalog_id=catalog_id,
-            table_service=self.table_service,
-        )
-        self.feature_service = FeatureService(
-            user=user,
-            persistent=persistent,
-            catalog_id=catalog_id,
-            table_service=self.table_service,
-            feature_namespace_service=FeatureNamespaceService(
-                user=user, persistent=persistent, catalog_id=catalog_id
-            ),
-            namespace_handler=NamespaceHandler(
-                view_construction_service=self.view_construction_service
-            ),
-        )
-        self.task_manager = TaskManager(
-            user=user, persistent=persistent, celery=celery, catalog_id=catalog_id
-        )
-        self.online_store_table_version_service = OnlineStoreTableVersionService(
-            user=user, persistent=persistent, catalog_id=catalog_id
-        )
-        self.tile_scheduler_service = TileSchedulerService(
-            user=user,
-            persistent=persistent,
-            catalog_id=catalog_id,
-            task_manager=self.task_manager,
-        )
-        self.tile_manager_service = TileManagerService(
-            user=user,
-            persistent=persistent,
-            catalog_id=catalog_id,
-            online_store_table_version_service=self.online_store_table_version_service,
-            tile_scheduler_service=self.tile_scheduler_service,
-        )
-        self.feature_manager_service = FeatureManagerService(
-            user=user,
-            persistent=persistent,
-            catalog_id=catalog_id,
-            tile_manager_service=self.tile_manager_service,
-        )
+        self.tile_scheduler_service = tile_scheduler_service
+        self.tile_manager_service = tile_manager_service
+        self.feature_manager_service = feature_manager_service
+        self.feature_service = feature_service
```

Comment:
All these services are now targeting a dummy user id. Just making a note here, will need to double check whether that will cause any issues when the services are used.